mlfinlab/labeling/trend_scanning.py

```diff
@@ -5,8 +5,53 @@
 
 import pandas as pd
 import numpy as np
+from numba import njit
 
-from mlfinlab.structural_breaks.sadf import get_betas
+
+@njit
+def calculate_t_values(subset, min_sample_length, step):  # pragma: no cover
+    """
+    For loop for calculating linear regression every n steps.
+
+    :param subset: (np.array) subset of indecies for which we want to calculate t values
+    :return: (float) maximum t value and index of maximum t value
+    """
+    max_abs_t_value = -np.inf  # Maximum abs t-value of b_1 coefficient among l values
+    max_t_value_index = None  # Index with maximum t-value
+
+    for forward_window in np.arange(min_sample_length, subset.shape[0], step):
+
+        y_subset = subset[:forward_window].reshape(-1, 1)  # y{t}:y_{t+l}
+
+        # Array of [1, 0], [1, 1], [1, 2], ... [1, l] # b_0, b_1 coefficients
+        x_subset = np.ones((y_subset.shape[0], 2))
+        x_subset[:, 1] = np.arange(y_subset.shape[0])
+
+        # Get regression coefficients estimates
+        xy_ = x_subset.transpose() @ y_subset
+        xx_ = x_subset.transpose() @ x_subset
+
+        #   check for singularity
+        det = np.linalg.det(xx_)
+
+        # get coefficient and std from linear regression
+        if det == 0:
+            b_mean = np.array([[np.nan]])
+            b_std = np.array([[np.nan, np.nan]])
+        else:
+            xx_inv = np.linalg.inv(xx_)
+            b_mean = xx_inv @ xy_
+            err = y_subset - (x_subset @ b_mean)
+            b_std = np.dot(np.transpose(err), err) / (x_subset.shape[0] - x_subset.shape[1]) * xx_inv  # pylint: disable=E1136  # pylint/issues/3139
```

Comment:
This issue is fixed with the updated pylint, the `# pylint: disable` comment can be removed.